### PR TITLE
Product Page: Add HP partner badge SVG

### PR DESCRIPTION
**Pull request by sumukhipandey**
Closes https://github.com/iFixit/ifixit/issues/47574

## QA
Similar to https://github.com/iFixit/react-commerce/pull/1481

### CODE REVIEW

**Review by masonmcelvain — APPROVED** [review]
CR ✌🏻

**Comment by masonmcelvain** [comment]
Hm the build fails complaining about our font awesome token. Seems unrelated to this PR but I don't know why it's failing for only this PR

image

**Comment by danielbeardsley** [comment]
Huh... I'm a little surprised this didn't affect the JS bundle. I would have expected it to grow at the least by the size of the svg file. It breaks some of my assumptions and worries me a little.

**Comment by masonmcelvain** [comment]
The SVG is 605 bytes. What's the minimum bundle size increase that would trigger the bundle analysis to show a diff?

**Comment by danielbeardsley** [comment]
> I don't know why it's failing for only this PR

Is this related to #1636 ?

**Comment by danielbeardsley** [comment]
> What's the minimum bundle size increase that would trigger the bundle analysis to show a diff?

image

Maybe it's comparing gzipped size and that's less than the 500-byte threshold.

**Comment by masonmcelvain** [comment]
> > I don't know why it's failing for only this PR
> 
> Is this related to #1636 ?

Yea probably, the problem is reproducible https://github.com/iFixit/react-commerce/pull/1647

**Comment by masonmcelvain** [comment]
Working on a fix for the preview deployment: https://github.com/iFixit/react-commerce/pull/1648

**Review by masonmcelvain — DISMISSED** [review]
CR ✌🏻

**Comment by masonmcelvain** [comment]
Now the react-commerce preview is failing because [Strapi](https://hp--add-oem-badge.govinor.com) for this branch isn't running for some reason 😞

**Comment by masonmcelvain** [comment]
dev_block ✌🏻 Let's change the height of the SVG to match the other OEM Partnership logos

image

image

**Review by masonmcelvain — DISMISSED** [review]
CR ✌🏻

**Comment by masonmcelvain** [comment]
image
image

Let's add a little vertical spacing to the HP svg. None of the other SVG's butt up against the edge of the container.

File: frontend/templates/product/sections/ProductOverviewSection/GenuinePartBanner.tsx [hunk 1]
```diff
@@ -26,6 +27,7 @@ const partnerCodeToComponentMap: { [key: string]: React.FC } = {
    // Don't have permission yet
    // google_pixel: Google,
    crucial: () => <Crucial px="2" />,
+   hp: Hp,
```
Comment (masonmcelvain) [inline frontend/templates/product/sections/ProductOverviewSection/GenuinePartBanner.tsx:30]:
```suggestion
   hp: () => <Hp py="2" />,
```

Comment (sumukhipandey) [reply frontend/templates/product/sections/ProductOverviewSection/GenuinePartBanner.tsx]: Resolved [b282e23](https://github.com/iFixit/react-commerce/pull/1646/commits/b282e230da537bb52ec8a765cab47a3aa6f2c4c1)
Thank you!

**Comment by sumukhipandey** [comment]
un_dev_block 🦋

**Pull request merged**